olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -709,14 +759,15 @@ spec:
       type: ValidatingAdmissionWebhook
       deploymentName: knative-openshift
       admissionReviewVersions:
+        - v1alpha1
         - v1beta1
```

Comment:
`admissionReviewVersions` have nothing to do with CRD version, so we shouldn't add v1alpha1 here.

Reply:
These versions tell the API server which request version the webhook supports, Knative webhooks support v1beta1 and v1 admission review versions

Reply:
Ok will remove that part.

Reply:
@pierDipi Actually reading [here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/):

```
API servers send the first AdmissionReview version in the admissionReviewVersions list they support. If none of the versions in the list are supported by the API server, the configuration will not be allowed to be created. If an API server encounters a webhook configuration that was previously created and does not support any of the AdmissionReview versions the API server knows how to send, attempts to call to the webhook will fail and be subject to the [failure policy](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy).
```
The failure policy we have is ignore so it makes sense to have that list of versions.

Reply:
> The failure policy we have is ignore so it makes sense why we v1beta1 is not failing so far. According to the above quote it does not make sense what we have there so far. 

v1beta1 is not supposed to fail (regardless of the policy), why do you think it should fail on it or why doesn't it make sense?

> Now that we have both v1alpha1 and v1beta1 this should be ok.

Nope, the subject of the paragraph is `admissionReviewVersions` and not CRD version, v1alpha1 is not supported by Knative webhooks so it should not be there.

Reply:
These are not knative webhooks it is Knative operator's webhook. The version requested at the admissionReview is via an object version that the webhook should support.

Reply:
Ok I guess you are refering to the k8s requirements to set [v1beta1 and v1 only](https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/request/admissionreview.go#L150). Ok will change.